docs/source/r-and-r-essentials.rst

```diff
@@ -33,10 +33,50 @@ Now you can easily share your new meta-package with friends and colleagues by up
   anaconda login
   anaconda upload path/to/custom-r-bundle-0.1.0-0.tar.bz2
 
-Your friends and colleagues now have access to your Custom-R-Bundle from any computer with the command:
+Your friends and colleagues now have access to your Custom-R-Bundle from any computer with the command::
 
-``conda install -c <your anaconda.org username> custom-r-bundle``
+  conda install -c <your anaconda.org username> custom-r-bundle
 
 For more information, see Christine Doig's blog post `Jupyter and conda for R <https://www.continuum.io/blog/developer/jupyter-and-conda-r>`_.
 
+Microsoft R Open
+================
+
+`Microsoft R Open (MRO) <https://mran.revolutionanalytics.com/download/mro-for-mrs/>`_ is available through conda for 64-bit Windows, 64-bit OS X, and 64-bit Linux, on the conda channel "mro".
+
+There are two ways to install MRO and other R packages. The first is to add the "mro" channel to your :doc:`.condarc configuration file<config>` and then use ``conda install r`` and use conda to install other packages. The second is to specify the "mro" channel on the command line each time you use the conda install command: ``conda install -c mro r``
+
+It is possible to install R packages through the Comprehensive R Archive Network (CRAN) and not conda. These R packages will install into the currently active conda environment.
```

Comment:
Yeah.  Note that MRO includes the checkpoint package which installs from MRAN instead of CRAN.
https://mran.revolutionanalytics.com/documents/rro/reproducibility/